docs/installation.rst

```diff
@@ -199,3 +199,24 @@ The Debug Toolbar currently doesn't support Django Channels or async projects.
 If you are using Django channels are having issues getting panels to load,
 please review the documentation for the configuration option
 :ref:`RENDER_PANELS <RENDER_PANELS>`.
+
+
+HTMX
+^^^^
+
+If you're using `HTMX`_ to `boost a page`_ you will need to add the following
+event handler to your code:
+
+.. code-block:: javascript
+
+    if (typeof htmx !== "undefined") {
+        htmx.on("htmx:afterSettle", function(detail) {
```

Comment:
This should check for the existence of `djdt`. People are likely to paste this in and use it on production.

Reply:
I don't know htmx that well but does it ever happen that `window.htmx` is undefined on such pages? 

Maybe `const h = window.htmx, d = window.djdt; if (h && d) { ... }`? This is all a matter of taste and a very personal opinion (not as a DJDT maintainer!) of course but I prefer `window.<var>` to the somewhat magic global `<var>`.

Reply:
I can make that change.

Reply:
I decided against creating constants and went with `window.htmx`. I did realize that we can't check `window.djdt` there since it may not be loaded in yet. So I pushed that check into the event handler, then wrapped it all with `{% if debug %}` to better signal that this should only be used when the toolbar is installed.